compiler/src/ammtojs.ts

```diff
@@ -55,16 +66,28 @@ const functionbodyToJsText = (fnbody: LPNode, indent: string) => {
         : 'undefined';
       outText += `r.emit('${name}', ${arg})\n`;
     } else if (statement.has('exits')) {
-      outText += `${statement.get('exits').t.trim()}\n`;
+      if (typeof mutationDirective[1] === 'boolean') {
```

Comment:
these `typeof` checks are kinda bleh - can `mutationDirective[1]` be an enum instead? Or check on `!== undefined` to be more explicit?